che-theia-plugins.yaml

```diff
@@ -1,5 +1,10 @@
 version: 1.0.0
 plugins:
+  - repository:
+      url: https://github.com/DonJayamanne/gitHistoryVSCode
+      revision: 0.6.14
```

Comment:
This needs to be a valid tag/branch/reference in the git repository, so in this case it should be `v0.6.14`

https://github.com/DonJayamanne/gitHistoryVSCode/tree/v0.6.14